src/Mouf/Security/UserService/UserService.php

```diff
@@ -159,7 +159,7 @@ public function login($login, $password) {
 	 */
 	public function loginWithoutPassword($login) {
 		// First, if we are logged, let's unlog the user.
-		if (!$this->byPassIsLogged && $this->isLogged()) {
+		if ((!$this->byPassIsLogged || !isset($this->byPassIsLogged)) && $this->isLogged()) {

```

Comment:
Ok, so you mean I should remove the isset() an declare the parameter in the class ?